app.rb

```diff
@@ -0,0 +1,134 @@
+require_relative 'student'
+require_relative 'teacher'
+require_relative 'book'
+require_relative 'rental'
+
+class App
+  def initialize
+    @books = []
+    @persons = []
+    @rentals = []
+  end
+
+  def app_options
+    puts '___________________________________________'
+    puts '1 - List all books'
+    puts '2 - List all people'
+    puts '3 - Create a person'
+    puts '4 - Create a book'
+    puts '5 - Create a rental'
+    puts '6 - List all rentals for a given person id'
+    puts '7 - Exit'
+    puts '___________________________________________'
+  end
+
+  def run
+    user_answer = 0
+    while user_answer != 7
+      app_options
+      user_answer = gets.to_i
+      selected_options(user_answer)
+    end
+  end
+
+  def selected_options(user_answer) # rubocop:disable Metrics/CyclomaticComplexity
+    case user_answer
+    when 1
+      list_all_books
+    when 2
+      list_all_people
+    when 3
+      create_person
+    when 4
+      create_book
+    when 5
+      create_rental
+    when 6
+      list_all_rentals_person_id
+    when 7
+      p 'Goodbye!'
+    end
+  end
+
+  def list_all_books
+    @books.each do |book|
+      print "Title: #{book.title}, Author: #{book.author}"
+    end
+  end
+
+  def list_all_people
+    @persons.each do |person|
+      puts "#{person.class} Name: #{person.name}, ID: #{person.id}, Age: #{person.age}"
+    end
+  end
+
+  def create_person # rubocop:disable Metrics/MethodLength
+    print 'Do you want to create a student (1) or a teacher (2)?[Input the number]: '
+    student_teacher = gets.to_i
+
+    print 'Age: '
+    age = gets.chomp
+    print 'Name: '
+    name = gets.chomp
+
+    case student_teacher
+    when 1
+      print 'Has parent permission? [Y/N]: '
+      parent_permission = gets.chomp
+      new_student = Student.new(name: name, age: age, parent_permission: parent_permission)
+      @persons.push(new_student)
```

Comment:
- [ ] Great work on the project, but when I create a student with including parent_permission then I get an error as you did not initialize the `classroom`. So please fix this issue
### Error
![sssssss](https://user-images.githubusercontent.com/81727988/194687532-e4e12531-e79b-4789-8b81-eb330fdd5ac2.png)

Reply:
Done ✔️  
I forgot to change the parent_permission to a boolean when I was delivering it to the new_student. But now is working. Thank you!

![image](https://user-images.githubusercontent.com/47539402/194689574-2c9035ca-d25f-4016-855e-2e9a5d444dd0.png)